k8s/example/Chart.yaml

```diff
@@ -33,3 +33,6 @@ dependencies:
 - name: agent+plg
   version: <AGENT_PLG_CHART_REV>
   repository: "file://./../agent+plg"
+- name: agent+lber
```

Comment:
k8s/example should only be used for agent chart.

Reply:
agent+lber dependency removed

Reply:
this may fail for lber. Please remove